CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
-## [*Ongoing*](https://github.com/pbrisbin/downgrade/compare/v6.3.0...master)
 
-- [NEW] Handling hard dependencies during downgrading
 
-## [*v6.3.0*](https://github.com/pbrisbin/downgrade/compare/v6.1.0...v6.3.0)
+## [v6.3.1](https://github.com/pbrisbin/downgrade/compare/v6.3.0...v6.3.1)
```

Comment:
I don't think we need this entry. This is all internal stuff that users wouldn't care about, no need for it in the CHANGELOG.

I also wonder if we should generally keep CHANGELOG out of PRs for now. It seems strange to couple the PRs to release cadence. We can (and should) do whatever PRs we want, then update the CHANGELOG for release at the end (which I'm happy to own myself).

Reply:
Yes I don't have a preference for this. I just wanted to keep the changelog up-to-date. I am fine with your suggestion of squash-merging the first two commits, and then merging the third one.

We can handle the fourth commit afterwards.

Reply:
I won't change anything else for now, but feel free to edit it as you find best.